_includes/tutorials/custom-connector/confluent/markup/dev/package-connector.adoc

```diff
@@ -0,0 +1,7 @@
+In your terminal, run:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/custom-connector/confluent/code/tutorial-steps/dev/package-connector.sh %}</code></pre>
++++++
```

Comment:
for your consideration: Add a note here to ignore the warnings for  `a filesystem-root relative reference`